### DISCUSSION TITLE
Assistance Required: Grammarly Benefit from GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by fuad78** [community/community#148221] [GitHub Education]
### Hi everyone,

Hello GitHub Education Team,

I hope this message finds you well. My GitHub Student Developer Pack was approved on December 28, 2024, but I have not yet been able to access the Grammarly Premium benefit as of December 31, 2024.

I have reviewed the benefits page and followed the steps to claim the offer, but it does not appear to be active in my account. Could you kindly assist me in resolving this issue or guide me on any additional steps I need to take?

Thank you for your support, and I look forward to your response.

Best regards,

**Comment by davevad93** [comment]
Hi @fuad78 ,

First of all, it seems that Grammarly is no longer offered in the `Student Developer Pack`, as it isn't listed in the [GitHub Student Developer Pack offers](https://education.github.com/pack), here you'll find all the current offers of the pack.

Secondly, what's the status of your application?

If the label is green and states "`Congrats! Your request for @fuad78 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @fuad78 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995
